### PR TITLE
Swagger now displays descriptions and info about parameters

### PR DESCRIPTION
**Pull request by Letaguator**
Added the generation of XML documentation, and enabled such that swagger reads the xml documentation and displays it accordingly.

### CODE REVIEW

File: backend/backend.csproj [hunk 1]
```diff
@@ -4,6 +4,9 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DocumentationFile>bin\Debug\netcoreapp3.1\backend.xml</DocumentationFile>
```
Comment (Letaguator) [inline backend/backend.csproj]: For the reviewers: this compile flag ensures that xml documentation is generated!

**Comment by AndreasJJ** [comment]
It's still missing description for GET and PUT at /api/Metadata, GET at /api/MetadataType, GET at /api/Municipality, GET and PUT at /api/Tag, POST GET PUT at /api/User/auth

**Comment by Letaguator** [comment]
So far no descriptions have been added yet, it currently only enables displaying of descriptions.

**Comment by AndreasJJ** [comment]
Might as well add them so that the pull request can be connected to #61 ?

**Comment by Letaguator** [comment]
Thought about that, but it can cause a conflict with the work I did in another branch, where I updated documentation on methods I reworked. See #72. So might be best to merge this, and add documentation when reworking methods. And do a more thorough commenting round after all of these Pull Requests/branches are merged?

**Comment by AndreasJJ** [comment]
Well, that's why you shouldn't change stuff that's outside the scope of the issue, but its whatever as long as its done before the sprint is over.

**Comment by Letaguator** [comment]
I know, since I changed the functionality it was within the scope on #72 as far as I know. I have now started adding documentation to all of the new functionality I add.

**Pull request merged**